lib/importproject.cpp

```diff
@@ -339,7 +339,30 @@ void ImportProject::importCompileCommands(std::istream &istr)
             dirpath += '/';
 
         const std::string directory = dirpath;
-        const std::string command = obj["command"].get<std::string>();
+
+        std::stringstream comm;
+        if( obj.find( "arguments" ) != obj.end() ) {
+            if( obj[ "arguments" ].is< picojson::array >() ) {
+                for( const picojson::value& arg : obj[ "arguments" ].get< picojson::array >() ) {
+                    if( arg.is< std::string >() ) {
+                        comm << arg.get< std::string >() << " ";
+                    }
+                }
+            }
+            else {
+                return;
+            }
+        }
+        else if( obj.find( "arguments" ) != obj.end() ) {
```

Comment:
/arguments/command/ ?

Reply:
Marvel at the power of cut and paste - whoops!